<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>office_sa</class>
 <widget class="QWidget" name="office_sa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#office_sa{background-image: url(:/hospital.jpg);}</string>
  </property>
  <widget class="QPushButton" name="b">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="office">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="stage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modify">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>480</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leader_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stage_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室主任</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
